### PR TITLE
Restore old CMP

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

adds `oldCmp` back so we can proceed with improvements to CMP while working out how to remove old cmp from dotcom

cc @ripecosta – back from the grave!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,9 @@
 		"types": "tsc --project ./tsconfig.build.json",
 		"validate": "npm-run-all --parallel tsc lint 'test --onlyChanged'"
 	},
+	"dependencies": {
+		"@guardian/old-cmp": "npm:@guardian/consent-management-platform@^3.4.10"
```
Comment (mxdvl) [inline package.json:31]: Be aware 3.4.10 is compatible with Source v2.0!

File: src/index.ts [hunk 2]
```diff
@@ -79,3 +79,5 @@ if (window) {
 export const { cmp, onConsentChange } =
 	(window?.guCmpHotFix as typeof actualExports) || actualExports;
 // *************** END commercial.dcr.js hotfix ***************
+
+export { oldCmp } from './oldCmp';
```
Comment (mxdvl) [inline src/index.ts:83]:
```suggestion
export { oldCmp } from './oldCmp'; // DEPRECATED: will be tree-shaken out
```

File: README.md [hunk 3]
```diff
@@ -155,6 +155,176 @@ onConsentChange(({ tcfv2, ccpa }) => {
 });
 ```
 
+## TCFv1 (deprecated)
+
+_These should not be used after 15 August 2020, and will be deleted shortly afterwards._
```
Comment (mxdvl) [inline README.md:160]:
```suggestion
_Since 15 August 2020, TCFv1 is deprecated and `oldCmp` should not be used._
```

**Review by mxdvl — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sndrs) [reply package.json:31]: this is the version currently shipping in `4.0.*` – we should probably match that?

Comment (mxdvl) [reply package.json:31]: Great!

**Pull request merged**